packages/desktop/components/popups/AllotManaPopup.svelte

```diff
@@ -24,28 +26,30 @@
     let error: string
     let amount: string
     let assetAmountInput: AssetAmountInput
-    let confirmDisabled = false
 
     const transactionInfo: ITransactionInfoToCalculateManaCost = {}
     let hasEnoughMana = false
 
     $: hasTransactionInProgress =
         $selectedWallet?.hasConsolidatingOutputsTransactionInProgress || $selectedWallet?.isTransferring
-    $: amount, accountAddress, hasTransactionInProgress, setConfirmDisabled()
     $: asset = $visibleSelectedWalletAssets[$activeProfile?.network?.id].mana
     $: validAmount = Big(rawAmount ?? 0)?.gt(0)
-    $: accountAddress, validAmount, rawAmount, void preparedOutput()
+    $: accountAddress, validAmount, void preparedOutput()
+    $: accountAddress, void validateAddress()
 
-    $: displayManaBox = !!accountAddress && !error && validAmount
+    $: submitAllowed =
```

Comment:
this is not working... :see_no_evil: t's allowing me to send even though I don't have a correct address :pray: 
![Screenshot from 2024-04-26 13-29-45](https://github.com/iotaledger/firefly/assets/72078237/e6656310-7346-4307-b0a3-ef32d9a4b7d3)